_content/articles/translation-of-pancagatidipani_hazlewood-a.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Translation of Pancagatidipani"
+authors:
+  - hazlewood-a
```

Comment:
Hazlewood doesn't have a `_authors` entry does she?

Reply:
I saw how you marked Hazelwood's name in another entry and thought it was a slug from the look of it. I fixed it.

Reply:
Oh, you mean the filename?  Yeah, I can be a bit lazy and abbreviate in the filename sometimes. I think for that one in particular, I was anticipating eventually adding an entry for her in the future. No worries 😊